spec/features/post_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../spec_helper'
+
+feature 'OpenSourceCMS Posts test', type: :feature do
+  describe 'When on Posts Page' do
+
+    before(:each) do
+      @posts_page = Posts.new
+      @posts_page.open_main_page
+    end
+
+    it 'Verifies possibility of adding new post by user' do
+      @posts_page.add_new_post("My test post")
```

Comment:
There are a lot of debates, but it is question of style and consistence, It is more common to use single quotes instead of double quotes if it is possible (you need double quotes in interpolation), more on this can be found here https://stackoverflow.com/questions/1836467/is-there-a-performance-gain-in-using-single-quotes-vs-double-quotes-in-ruby